text/1201-naked-fns.md

```diff
@@ -3,6 +3,8 @@
 - RFC PR: [rust-lang/rfcs#1201](https://github.com/rust-lang/rfcs/pull/1201)
 - Rust Issue: [rust-lang/rust#32408](https://github.com/rust-lang/rust/issues/32408)
 
+> ⚠ This RFC has been superseded by RFC 2972, constrained naked functions. ⚠
```

Comment:
Can we add some links here?  For example, <https://github.com/rust-lang/rfcs/pull/3073/files#diff-58590e5b7b8136b87d9f58b95bea9ad2a1e5aee4b756bddcc69023ea7ef09f39R10> linked to the other RFC and to the FCP where the decision was made.

Otherwise r=me.